website/docs/docs/dbt-versions/release-notes/04-Sept-2023/ci-updates-phase2-rn.md

```diff
@@ -29,4 +29,8 @@ Below is a comparison table that describes how deploy jobs and CI jobs behave di
 
 ## What you need to update
 
-If you previously set up a job using the [Create Job](/dbt-cloud/api-v2#/operations/Create%20Job) API endpoint before September 11, 2023, you must re-create the job as described in [Trigger a CI job with the API](/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api). This is because you must set the `job_type` to be `ci`. 
\ No newline at end of file
+- If you want to set up a CI environment for your jobs, dbt Labs recommends that you create your CI job in a dedicated [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) that's connected to a staging database. To learn more about these environment best practices, refer to the guide [Get started with continuous integration tests](/guides/orchestration/set-up-ci/overview).
+
+- If you previously set up a job using the [Create Job](/dbt-cloud/api-v2#/operations/Create%20Job) API endpoint before September 11, 2023, you must re-create the job as described in [Trigger a CI job with the API](/docs/deploy/ci-jobs#trigger-a-ci-job-with-the-api). This is because you must set the `job_type` to be `ci`. 
+
+- If you previously set up a job using dbt Cloud before September 11, 2023, you must re-create the job as described in [Set up CI jobs](/docs/deploy/ci-jobs#set-up-ci-jobs) and make sure to enable the option **Triggered by pull requests**.  
```

Comment:
post-rollout, they will be able to recreate the job in the UI by selecting to create CI job; they won't necessarily need to enable PR triggers

Reply:
good point, will reword this, thanks!